tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -101,6 +101,16 @@ def wait_for_app_update_to_clear(self):
         Wait(self.marionette).until(lambda m: element.location['y'] == (0 - element.size['height']))
 
     @property
+    def screen_width(self):
+        self.marionette.switch_to_frame()
+        return self.marionette.find_element(*self._screen_locator).rect['width']
+
```

Comment:
I guess screen_width also could be done with screen.width * window.devicePixelRatio.
Then the switch_to_frame call is not necessarily needed, perhaps.